src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -105,99 +105,149 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
 
-    "Failing due to Thurloe error should return a server error to the user" in {
-      Put(userAgreementPath) ~> dummyUserIdHeaders(dummy1User) ~> userServiceRoutes ~> check {
-        status should equal(InternalServerError)
-        assert(localThurloeDao.agreedUsers.isEmpty)
+      "failing due to Thurloe error should return a server error to the user" in {
+        Put(userAgreementPath) ~> dummyUserIdHeaders(dummy1User) ~> userServiceRoutes ~> check {
+          status should equal(InternalServerError)
+          assert(localThurloeDao.agreedUsers.isEmpty)
+        }
       }
-    }
 
-    allHttpMethodsExcept(PUT) foreach { method =>
-      s"should reject ${method.toString} method" in {
-        new RequestBuilder(method)(userAgreementPath) ~> dummyUserIdHeaders(enabledUser) ~> userServiceRoutes ~> check {
-          assert(!handled)
-          assert(localThurloeDao.agreedUsers.isEmpty)
+      allHttpMethodsExcept(PUT) foreach { method =>
+        s"should reject ${method.toString} method" in {
+          new RequestBuilder(method)(userAgreementPath) ~> dummyUserIdHeaders(enabledUser) ~> userServiceRoutes ~> check {
+            assert(!handled)
+            assert(localThurloeDao.agreedUsers.isEmpty)
+          }
         }
       }
-    }
 
-    allButEnabledUsers foreach { user =>
-      s"$user should not be able to agree to terms" in {
-        Put(userAgreementPath) ~> dummyUserIdHeaders(user) ~> userServiceRoutes ~> check {
-          status should equal(Forbidden)
-          assert(localThurloeDao.agreedUsers.isEmpty)
+      allButEnabledUsers foreach { user =>
+        s"should not allow a $user" in {
+          Put(userAgreementPath) ~> dummyUserIdHeaders(user) ~> userServiceRoutes ~> check {
+            status should equal(Forbidden)
+            assert(localThurloeDao.agreedUsers.isEmpty)
+          }
         }
       }
-    }
 
-    "attempting to agree to terms as enabled user" in {
-      Put(userAgreementPath) ~> dummyUserIdHeaders(enabledUser) ~> userServiceRoutes ~> check {
-        status should equal(NoContent)
-        assert(localThurloeDao.agreedUsers == Seq("enabled-user"))
+      "attempting to agree to terms as enabled user" in {
+        Put(userAgreementPath) ~> dummyUserIdHeaders(enabledUser) ~> userServiceRoutes ~> check {
+          status should equal(NoContent)
+          assert(localThurloeDao.agreedUsers == Seq("enabled-user"))
+        }
       }
     }
-  }
 
-  "Free Trial Enrollment" - {
-    val enrollPath = "/api/profile/trial"
+    val trialPathBase = "/api/profile/trial"
 
-    "User-initiated enrollment endpoint" - {
-      allHttpMethodsExcept(POST) foreach { method =>
-        s"should reject ${method.toString} method" in {
-          new RequestBuilder(method)(enrollPath) ~> dummyUserIdHeaders(enabledUser) ~> userServiceRoutes ~> check {
-            assert(!handled)
+    val enrollPaths = Seq(trialPathBase, s"$trialPathBase?operation=enroll")
```

Comment:
it's worth adding a test without the `operation` param to check the default behavior.

Reply:
By 'without the `operation` param', do you mean `/api/profile/trial`? If so, `trialPathBase` covers that.

Reply:
yes, you're right. I misread, my bad!